views/albums.hbs

```diff
@@ -0,0 +1,7 @@
+<section id="albums">
+{{#each albums}}
+<h1>{{this.name}}</h1>
```

Comment:
On the lab instructions they ask you to also show the image of the album in the albums page:

```
<img src="{{this.images.0.url}}" alt="image">
```

It's the same as you did in the artist-search-results page.